ccf_transaction_processor/scripts/start_cchost.sh

```diff
@@ -28,20 +28,9 @@ if [ $SGX_MODE == "SIM" ]; then
    CCHOST=${F_SERVICEHOME}/bin/cchost.virtual
 fi
 
-if [ -f ${F_SERVICEHOME}/run/cchost.pid ]; then
-    if ps -p $(cat ${F_SERVICEHOME}/run/cchost.pid) > /dev/null
-    then
-        yell cchost appears to be running already
-        exit -1
-    fi
-fi
-
-rm -f ${F_SERVICEHOME}/run/*
-rm -f ${F_SERVICEHOME}/logs/*.log
-
```

Comment:
Thank you for the question. the same check is present in start_ccf_network.sh, and given that start_cchost is only invoked from within start_ccf_network.sh, one of the two checks is redundant. remove it here,  since it made sense to check it even earlier in the overall deployment.

Reply:
why do you say so that it is used only from start_ccf_network?  If this is really the only allowed use, then i don't think start_cchost.sh should exist as a standalone script.  That said, once you have a network running, doing for individual hosts issue stop and start should be perfectly legimate, so having a separate script makes sense (but then it should do error checking)

Reply:
the reason it is a separate script is that, once we go to multi-node, the start_cchost will also be called by the add_node_script. (which is not there in this PR). Both the sstart_ccf_network and add_node_script are essentially wrappers around start_cchost that in addition to starting a node does additional configuration steps suitable for the start or add. (like creating ledger authority for start, and "adding a node as a trusted node" for add, etc). Could have merged the two into one script with lots of "if-else" but that seems too ugly, and no reason to do that. 

Like I said earlier, I can always move the error checking from the wrapper script to inside start_cchost, I see no reason to favor one over the other for the recommended usecases.

Reply:
given that you already use script libraries with common.sh and if the functionality is only used as subroutine,   then defining as shared functionality in a library which is not in `bin` would seem a bit more fail-safe? At least to me something which is in `bin` is a standalone command.   But obviously a matter of taste ...

Reply:
@g2flyer remember that those routines are not currently installed with the CCF installation. and they would need to be further split in order to function correctly for the ccf. i've added issued #295 to address this.

Reply:
@cmickeyb i was referring to the `source ${F_SERVICEHOME}/bin/lib/pdo_common.sh` which `start_cchost.sh` _already_ does.  But given that we do not check errors, maybe nobody complained that the file didn't exist ;-)